src/Components/CroppedThumbnail/CroppedThumbnail.spec.tsx

```diff
@@ -0,0 +1,38 @@
+import { render } from "@testing-library/react-native";
+import * as React from "react";
+import CroppedThumbnail from "./CroppedThumbnail";
+
+it("should render CroppedThumbnail", async () => {
+  const imageId = "dk7o2yxitrwcsjz98fvk";
+  const { findByLabelText } = render(
+    <CroppedThumbnail imageId={imageId} options={{ width: 200 }} accessibilityLabel="image" />,
+  );
+  const image = await findByLabelText("image");
+  expect(image).toMatchInlineSnapshot(`
+<Image
+  accessibilityLabel="image"
+  accessible={true}
+  imageId="dk7o2yxitrwcsjz98fvk"
```

Comment:
You are passing an unknown props to `Image` component.

You should get a warning like this: https://reactjs.org/warnings/unknown-prop.html